includes/class-wcmp-settings-callbacks.php

```diff
@@ -25,17 +25,44 @@ public function section() { }
      *   description - description (optional)
      * @return void.
      */
-    public function checkbox($args) {
+    public function checkbox($args, $disabled = '', $style = '') {
         extract($this->normalize_settings_args($args));
 
+        // check if age_check is active and disabled moning and evening delivery
+        if (isset(WooCommerce_MyParcel()->export_defaults['age_check']) &&
+            (
+                $id == "morning_enabled" ||
+                $id == "evening_enabled" ||
+                $id == "only_recipient_enabled" ||
+                $id == "signature_enabled"
+            )){
+
+            $current = 0;
+            $disabled = 'disabled';
+            $style = 'cursor: not-allowed;';
+            $description = __("Not possible if age check is active", 'woocommerce-myparcel');
+
+            if ($id == "only_recipient_enabled"){
+                $current = 1;
+                $description = __("Home address only is mandatory when age check is active", 'woocommerce-myparcel');
+            }
+
+            if ($id == "signature_enabled"){
+                $current = 1;
+                $description = __("Signature for receipt is mandatory when age check is active", 'woocommerce-myparcel');
+            }
+        }
+
         // output checkbox
         printf(
-            '<input type="checkbox" id="%1$s" name="%2$s" value="%3$s" %4$s class="%5$s"/>',
+            '<input type="checkbox" id="%1$s" name="%2$s" value="%3$s" %4$s class="%5$s" %6$s style="%7$s"/>',
             $id,
             $setting_name,
             $value,
             checked($value, $current, false),
-            $class
+            $class,
+            $disabled,
+            $style
```

Comment:
```suggestion
            $style,
```

Reply:
Dit werkt niet met een printf hierdoor zal de volgende error te zien zien
` Parse error: syntax error, unexpected ')' in /woocommerce-myparcel/includes/class-wcmp-settings-callbacks.php on line 65`

Reply:
Oh duh, dacht dat het een array was :P nvm

Reply:
ook als het een array was, dan mag je hier geen komma gebruiken (we moeten voor wc php versie 5.4 ondersteunen)